app.js

```diff
@@ -6,21 +6,31 @@ const Controller = require('./MVC/controller');
 const app = express();
 const apiRouter = express.Router();
 const topicsRouter = express.Router();
+const articlesRouter = express.Router();
 
 const model = new Model(db);
 const controller = new Controller(model);
 
 app.use(express.json());
```

Comment:
We do not need express.json() until we move onto a patch or post, please remove.